src/core/ReactInstanceMap.js

```diff
@@ -18,6 +18,9 @@
  * to internal methods.
  */
 
+var internalKey = '_reactInternal' +
+                  (Math.random().toString().substr(2, 7));
```

Comment:
Let's indent like this instead:

```js
 var internalKey =
  '_reactInternal' + Math.random().toString().substr(2, 7);
```

Also, should we make it a CONSTANT (so it's a bit easier to one day switch ti using `const`)?